apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph.scss

```diff
@@ -0,0 +1,3 @@
+:host ::ng-deep button {
+  font-weight: 600;
```

Comment:
this will give all buttons within the decisiongraph a font-weight of 600. 
Is this what we want or do we specifically target buttons within the ba-decision-graph components own template?

Reply:
Yea, every button needs to have a font-weight of 600.